notebooks/finetune/Flan-T5-3B/RosaQA.ipynb

```diff
@@ -0,0 +1,7957 @@
+{
```

Comment:
<p>showing <code>rosa_dt</code> here seems redundant? (it is shown above when loading the dataset, and it is unchanged so far)</p><p> </p><p>Maybe we might want to show how the dataset evolves with tokenization? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>